Vungle/src/main/java/com/mopub/mobileads/VungleAdapterConfiguration.java

```diff
@@ -42,8 +48,13 @@ public String getAdapterVersion() {
     @Nullable
     @Override
     public String getBiddingToken(@NonNull Context context) {
-        final String bidToken = Vungle.getAvailableBidTokens(10);
-        return bidToken;
+        if (initBidToken.compareAndSet(false, true)) {
+            String token = Vungle.getAvailableBidTokens(10);
```

Comment:
@palrahul workaround for fix getting bidToken for the first time. We can move it into initializeNetwork() method when SDK changes approved in the future.

Reply:
@HuFangshuai can you help me understand the benefit this initBidToken check provides. We can get the getBidToken call before init, and in that case getAvailableBidTokens will still return null, so in that case should we set the initBidToken to true  ?

Reply:
have changed to always run on main thread if haven't got the bidToken.

Reply:
@HuFangshuai this still has issue of possibly returning old super token on each request,  do we really need the Thread in which we are calling getAvailableBidTokens. , we already have logic inside SDK to avoid ANRs. If we remove the execution in thread it should. at least try to run main thread. and get the supertoken, and if it returns null then we can try to use the older in-mem copy(if available). Seems removing thread and keeping all other logic as-is it should cover majority of scenarios we are trying to address.

Reply:
Yes, make sense.